### PR TITLE
EAHW-2122 Deploy timecard API

### PR DESCRIPTION
**Pull request by sewerynmi**

### CODE REVIEW

File: helm/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: callisto-timecard-restapi
+description: A Helm chart for deploying Callisto Timecard REST api
+type: application
+version: 1.0.0
+appVersion: 1.0.0
```
Comment (laurieboyes) [inline helm/Chart.yaml:6]: I’m not sure what the implications of these versions are 🤔 I feel like we’re definitely not at version 1.0.0 though. 

[This site suggests that appVersion is optional](https://helm.sh/docs/topics/charts/) and it seems fairly meaningless so maybe we can delete this line?

For version it doesn’t _seem_ like it’s of much consequence ([according to this site at least](https://codefresh.io/docs/docs/new-helm/helm-best-practices/#chart-versions-and-appversions)) so maybe 1.0.0 is fine if that’s what we’re doing elsewhere 🤷‍♀️

```suggestion
```

**Review by laurieboyes — APPROVED** [review]
Nice 👍 one _very_ small comment

File: helm/Chart.yaml (hunk 1 above)
Comment (joannafawl) [reply helm/Chart.yaml:6]: Happy to delete `appVersion`. I'm not sure about `version` either, I think it's fine to keep it at `1.0.0` for now but not sure if we should be updating the version from now on every time we make a change to the helm files?

Comment (laurieboyes) [reply helm/Chart.yaml]: Doesn’t seem that way. Seems pretty arbitrary from a brief google 🤷

**Review by laurieboyes — APPROVED** [review]

**Review by joannafawl — APPROVED** [review]

**Pull request merged**